src/VPolygon.jl

```diff
@@ -585,8 +585,10 @@ function minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
     mP = length(vlistP)
     mQ = length(vlistQ)
     i = 1
-    k = 1
-    j = 1
+    σP = σ(N[1, 0], P)
+    σQ = σ(N[1, 0], Q)
+    k = findfirst(==(σP), vlistP)
```

Comment:
:warning: Equality check doesn't play well with floating point. In this case it won't hurt since we get a vertex from `VPolygon`'s list of vertices [here](https://github.com/JuliaReach/LazySets.jl/blob/master/src/VPolygon.jl#L264) by asking the support vector, but this is very specific to polygons.

So i think a cleaner (and slightly more efficient) approach would be to get directly the corresponding index from `VPolygon`'s support vector function. It suffices to outsource the algorithm in https://github.com/JuliaReach/LazySets.jl/blob/master/src/VPolygon.jl#L256 to an independent helper function.